pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -76,6 +76,14 @@ type BundleSource struct {
 	// InLine is a simple string to append as the source data.
 	// +optional
 	InLine *string `json:"inLine,omitempty"`
+
+	// DefaultPackage, when true, requests the default bundle package to be used as a source.
```

Comment:
I feel like the naming ('DefaultPackage') suggests that these CA certs will be included _by default_ in any Bundle even when this is not specified, which is not the case.
I also think that 'package' is not sufficiently descriptive.

I think that naming this source type in a way that conveys where it actually comes from (CAs compiled by cert-manager in this case) would be more valuable. Perhaps the source type could be named just 'CertManager'? It would also make it easier to refer to it in other contexts by calling it 'cert manager trust source' which is in my opinion more descriptive than 'default package'.

Reply:
What do you think to the names Tim and I discussed on Friday? E.g. https://github.com/cert-manager/trust-manager/pull/46#discussion_r1050906535

Reply:
I would be against using a verb like 'include' for the same reason that the 'secret' source is not called 'includeFromSecret'.

But I also think that any version of these source names should be okay given that this is `v1alpha1` API version and we could rename it later if the users complain.

Reply:
I see your point re verbs, but I think I'm actually kinda OK with using a verb for this specific field because it's a boolean. It wouldn't make sense for `secret` or `configMap`, I agree.

`useDefaultCAs` feels fine though because it's a yes / no (true / false) question. "Do you want to use the default CAs?". It also gets around the idea that the CA certs will be included by default I think.

I'm a little loathe to use something like `certManager` because anyone could configure this to point to any bundle by setting `--default-package-location=/my/bundle.json` - in which case it wouldn't necc. have anything to do with cert-manager. 

- "What are the default CAs if I installed with Helm and didn't change anything?" - cert-manager debian bundle
- "What are the default CAs if I installed with Helm and changed the file it points to?" - Whatever you pointed it at
- "What are the default CAs if I installed manually?" Not available unless you set `--default-package-location`

Reply:
> because anyone could configure this to point to any bundle by setting --default-package-location=/my/bundle.json

I didn't think about this, thanks 👍🏼 

In that case I do prefer `useDefaultCAs` to `DefaultPackage` because it does not introduce a potentially unneccesary extra term (`package`)

Reply:
Yeah I think we agree then! I think package makes sense on the backend - the physical JSON bundle can be called a "package" - but at the point of consumption I think "package" is more confusing!